### ISSUE TITLE
Update canvas2pl to handle "NewQuizzes" questions

### ISSUE THREAD

**Issue by firasm** [PrairieLearn/PrairieLearn#10087]
[There's a script](https://github.com/PrairieLearn/PrairieLearn/tree/master/tools/question_converters/canvas) (thanks Jonatan!) that converts Classic Canvas questions to PrairieLearn questions.

This summer I'm working on a course that would like to switch to PL from Canvas, but they're using "New Quizzes" and we'd like to update the script to handle those quiz formats.

Creating an issue to track this.

**Comment by jonatanschroeder** [comment]
I know I've been mentioned in the issue, but I don't work with Canvas here at York, so I think it's unlikely I'd take over this. I'd encourage you to get someone (a student?) to take care of this. A brief check in the Canvas API documentation seems to indicate that translating the code from [the old quiz model](https://canvas.instructure.com/doc/api/quizzes.html) to [the new one](https://canvas.instructure.com/doc/api/new_quizzes.html) may be reasonable.

**Comment by firasm** [comment]
Yep all good @jonatanschroeder I just created the issue because I have someone who'll be working on this in the next few days/weeks

**Comment by trombonekenny** [comment]
I can help you with this. I have some New Quiz questions I'd like to try converting so I can be another test case.

**Comment by jonatanschroeder** [comment]
@firasm I believe new quizzes can be exported to QTI, so can this be closed?

**Comment by firasm** [comment]
Yup! this can be closed now, thanks!
